app/server/lib/sendAppPage.ts

```diff
@@ -98,6 +99,7 @@ export function makeGristConfig(options: MakeGristConfigOptions): GristLoadConfi
     survey: Boolean(process.env.DOC_ID_NEW_USER_INFO),
     tagManagerId: process.env.GOOGLE_TAG_MANAGER_ID,
     activation: (req as RequestWithLogin|undefined)?.activation,
+    latestVersionAvailable: (server as FlexServer)?.getLatestVersionAvailable(),
```

Comment:
You should add those methods to the `GristServer` interface. If you grep for `import.*FlexServer`, you will see we almost never use this server directly (and when it is used, it is either in the `app entrypoint` or a mistake (like in plugins)).

Reply:
Makes sense. I did this originally because I had a silly type error when using `GristServer`, but I've figured out how to fix it.

Moved the methods to `GristServer`.